omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/model/AdultHivFlowsheetFormData.java

```diff
@@ -256,7 +257,7 @@ private void addARVRegimenTableRow(StringBuilder builder, List<DrugOrder> drugOr
     	
     	builder.append("</tr>\n");
     	arvRowNum++;
-		//<lookup complexExpression="#if($fn.getDrugOrders(''))#set($drugOrders = $fn.getDrugOrders('ANTIRETROVIRAL DRUGS'))#foreach($drugOrder in $drugOrders) &lt;tr&gt; &lt;td align=&quot;center&quot;&gt; &#x2713; &lt;/td&gt;     &lt;td&gt; Commencer: $!{fn.formatDate('$FormatDate_General', $drugOrder.getStartDate())} &lt;br/&gt;$!{drugOrder.getDrug().getName()} $!{drugOrder.getDose()}$!{drugOrder.getUnits().replaceAll('tab\(s\)', 'ce')} $!{drugOrder.getFrequency().replaceAll(' x 7 days/week', '').replaceAll('day', 'j')}&lt;/td&gt; #if($drugOrder.getDiscontinued())&lt;td align=&quot;center&quot;&gt; &#x2713; &lt;/td&gt;  &lt;td&gt; Arret: $!{fn.formatDate('$FormatDate_General', $drugOrder.getDiscontinuedDate())} &lt;br/&gt;Raison: $!{drugOrder.getDiscontinuedReason().getName()} &lt;/td&gt; #else &lt;td&gt;  &lt;/td&gt;  &lt;td&gt; Arret: $DateTextPlaceHolder &lt;br/&gt;Raison:  &lt;/td&gt; #end &lt;/tr&gt;#end#end" />
+		//<lookup complexExpression="#if($fn.getDrugOrders(''))#set($drugOrders = $fn.getDrugOrders('ANTIRETROVIRAL DRUGS'))#foreach($drugOrder in $drugOrders) &lt;tr&gt; &lt;td align=&quot;center&quot;&gt; &#x2713; &lt;/td&gt;     &lt;td&gt; Commencer: $!{fn.formatDate('$FormatDate_General', $drugOrder.getEffectiveStartDate())} &lt;br/&gt;$!{drugOrder.getDrug().getName()} $!{drugOrder.getDose()}$!{drugOrder.getUnits().replaceAll('tab\(s\)', 'ce')} $!{drugOrder.getFrequency().replaceAll(' x 7 days/week', '').replaceAll('day', 'j')}&lt;/td&gt; #if($drugOrder.getDiscontinued())&lt;td align=&quot;center&quot;&gt; &#x2713; &lt;/td&gt;  &lt;td&gt; Arret: $!{fn.formatDate('$FormatDate_General', $drugOrder.getEffectiveStopDate())} &lt;br/&gt;Raison: $!{drugOrder.getDiscontinuedReason().getName()} &lt;/td&gt; #else &lt;td&gt;  &lt;/td&gt;  &lt;td&gt; Arret: $DateTextPlaceHolder &lt;br/&gt;Raison:  &lt;/td&gt; #end &lt;/tr&gt;#end#end" />
```

Comment:
No idea here.  I doubt this is going to work as intended, and probably needs to be independently ticketed for someone to look into/address as part of the order entry work overall.

Reply:
I'll test it on a 1.9 instance and see how it behaves and once we merge this PR, I'll test on a 2.3 instance to compare and create a ticket if things break.